runtime/connections.go

```diff
@@ -199,11 +199,20 @@ func (r *Runtime) ConnectorConfig(ctx context.Context, instanceID, name string)
 		}
 	}
 
+	vars := inst.ResolveVariables()
 	// Search for connector definition in rill.yaml
 	for _, c := range inst.ProjectConnectors {
```

Comment:
nit: don't put a line of code above a comment that covers a whole section of code (put the code under the comment or add a newline above the comment)